src/books.js

```diff
@@ -130,7 +128,6 @@ booksArray.forEach((book) => {
 
   book.readingTime = readingTime;
 
-  console.log(booksArray);
```

Comment:
The instructions called for logging your booksArray. I didn't say remove them. I said this particular one was in the wrong place.